src/lib/application.py

```diff
@@ -294,10 +314,26 @@ def is_analysis_complete(self):
         Returns:
             (bool): True if analysis was completed
         """
-        self.wait_find_element(
-            locator_type="image", locator="run_complete.png", timeout=120.0, interval=5.0,
-        )
-        return True
+        try:
+            self.wait_find_element(
+                locator_type="image", locator="report_page_header.png", timeout=120.0, interval=5.0,
+            )
+            return True
+        except Exception:
+            return False
+
+    def verify_story_points(self):
+        """
+        Verifies the story points in report after analysis
+
+        Returns:
+            (bool): True if story points were accurate
+        """
+        try:
+            self.wait_find_element(locator_type="image", locator="story_points.png")
```

Comment:
It is checking for story point image right ? How can we validate if the story point is a number and same for all IDE's

Reply:
Yes, this is specifically, checking for acemair project, which generates 2 story points. It is being validated against image (that has 2 story points from web report) So, we need not worry about it being a number, if story points are generated other than 2, test will fail. Overall, this validation will always apply right check. And yes, this is same for all IDEs, since project is same